var/spack/repos/builtin/packages/gene-expression-volume/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class GeneExpressionVolume(PythonPackage):
+    """BBP command line interface for the re-alignment of 2D images and
+    the creation of 3D gene expression volumes.
+    Used by the atlas building pipeline.
+    """
+    homepage = "https://bbpgitlab.epfl.ch/nse/gene-expression-volume"
+    git      = "git@bbpgitlab.epfl.ch:nse/gene-expression-volume.git"
+
+    version('0.1.0', tag='gene-expression-volume-v0.1.0')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-atlalign@0.6.0', type=('build', 'run'))
+    depends_on('py-click@7.0:', type=('build', 'run'))
+    depends_on('py-nptyping@1.0.1:', type=('build', 'run'))
+    depends_on('py-numpy@1.15.0:', type=('build', 'run'))
+    depends_on('py-scikit-image@0.017.2:', type=('build', 'run'))
```

Comment:
Minor and probably working in either way:
```suggestion
    depends_on('py-scikit-image@0.17.2:', type=('build', 'run'))
```

Reply:
To be fixed in the next PR.